openstack/identity/v3/projects/requests.go

```diff
@@ -32,6 +32,18 @@ type ListOpts struct {
 	// ParentID filters the response by projects of a given parent project.
 	ParentID string `q:"parent_id"`
 
+	// Tags filters on specific project tags. All tags must be present for the project.
+	Tags *[]string `q:"tags"`
+
+	// TagsAny filters on specific project tags. At least one of the tags must be present for the project.
+	TagsAny *[]string `q:"tags-any"`
+
+	// NotTags filters on specific project tags. All tags must be absent for the project.
+	NotTags *[]string `q:"not-tags"`
+
+	// NotTagsAny filters on specific project tags. At least one of the tags must be absent for the project.
+	NotTagsAny *[]string `q:"not-tags-any"`
```

Comment:
Why are these pointers?

Reply:
I reviewed the ListOpt of the other resources and aligned project to them: so I removed the * and changed the array to a plain string. Sorry for the error.

Reply:
No problem at all!